docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -383,23 +266,28 @@ commodore catalog compile ${CLUSTER_ID} --push -i
 
 . Configure Terraform secrets
 +
-[source,console]
+[source,bash]
 ----
-cat <<EOF > catalog/manifests/openshift4-cloudscale/.env
+cat <<EOF > catalog/manifests/openshift4-terraform/terraform.env
```

Comment:
```suggestion
cat <<EOF > ./terraform.env
```

Depending on how you want to go with the Terraform alias.

Reply:
At this point, we have not yet invoked `pushd catalog/manifests/openshift4-terraform/`, so the path should stay.

Reply:
If you adjust the instructions as suggested in https://github.com/appuio/openshift4-docs/pull/96#discussion_r672251010 this change is required. The point would be to keep the `terraform.env` outside the cluster catalog so we don't have to recreate it after the catalog is recompiled.

Reply:
ah I see what it's about